core/templates/dev/head/css/oppia.css

```diff
@@ -3657,7 +3657,7 @@ md-card.preview-conversation-skin-supplemental-card {
 .oppia-about-header {
   background-blend-mode: multiply;
   background-color: #afd2eb;
-  background-image: url("../images/general/icons-bg.png");
+  background-image: url("../assets/common/images/general/icons-bg.png");
```

Comment:
Just wondering, do you know why this url has a `..` before it and the one above doesn't?